src/main/java/duke/logic/commands/results/Taskable.java

```diff
@@ -0,0 +1,10 @@
+package duke.logic.commands.results;
+
+import duke.model.TaskList;
+
+public interface Taskable {
```

Comment:
What is taskable?

Reply:
We are unsure of how to name the interface, it is to mean the class that implements the interface should contains tasks and a method to get the tasks.